llama_stack/distribution/utils/image_types.py

```diff
@@ -4,10 +4,10 @@
 # This source code is licensed under the terms described in the LICENSE file in
 # the root directory of this source tree.
 
-from enum import Enum
+import enum
 
 
-class ImageType(Enum):
-    container = "container"
-    conda = "conda"
-    venv = "venv"
+class LlamaStackImageType(enum.StrEnum):
```

Comment:
StrEnum is only available in Python >= 3.11 https://docs.python.org/3/library/enum.html#enum.StrEnum

Reply:
Oop I didn't see that in the docs, I'll make that backward compatible. Looks like we target `python>=3.10`